config/routes.rb

```diff
@@ -3,8 +3,6 @@
   resources :doctors
   root "users#index"
   resources :users
-  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
-
-  # Defines the root path route ("/")
-  # root "articles#index"
+  post "/login", to: "sessions#login"
+  post "/logout", to: "sessions#logout"
```

Comment:
Please add a sign_up routes.
  You can add this line after line 8: post "/sign_up", to: "users#create"